recipes/opencv/4.x/conanfile.py

```diff
@@ -283,62 +283,33 @@ def package(self):
         if os.path.isfile(os.path.join(self.package_folder, "setup_vars_opencv4.cmd")):
             os.rename(os.path.join(self.package_folder, "setup_vars_opencv4.cmd"),
                       os.path.join(self.package_folder, "res", "setup_vars_opencv4.cmd"))
+        self._create_cmake_module_alias_targets(
+            os.path.join(self.package_folder, self._module_subfolder, self._module_file),
+            {component["target"]:"opencv::{}".format(component["target"]) for component in self._opencv_components}
+        )
+
+    @staticmethod
+    def _create_cmake_module_alias_targets(module_file, targets):
```

Comment:
I dislike how much this is being copied around, it really need to be added to Conan

Reply:
Do there exist examples of this pattern for creating imported executables?

Reply:
I tried something in geographiclib but canceled the modification: https://github.com/conan-io/conan-center-index/pull/4295/commits/ec87edc7f023e156699fd4d17afdd14445aa3469

imported executables will be implemented in conan.